common_benchmark_suite/openxla/benchmark/comparative_suite/jax/scripts/generate_model_artifacts.py

```diff
@@ -102,10 +108,18 @@ def _parse_arguments() -> argparse.Namespace:
                       type=pathlib.Path,
                       default=None,
                       help="Path to `iree-opt`. Used to binarize mlir.")
+  parser.add_argument(
+      "--auto-upload",
+      "--auto_upload",
+      action=argparse.BooleanOptionalAction,
```

Comment:
Can we use `action="store_true"`?

Reply:
From Python 3.8, BooleanOptionalAction has the same effect as `action="store_true" `for `--auto-upload` and `action="store_false"` for `--no-auto-upload`. Shall we adopt this new version?

Reply:
I think `argparse.BooleanOptionalAction` is added in Python 3.9 (also I couldn't find it in 3.8 docs)

![image](https://github.com/openxla/openxla-benchmark/assets/2104162/4c0f3beb-aa4c-4f1d-86e5-dc937d616140)

I totally agree it's better but unfortunately we still pin at 3.8 ...

Reply:
Oh I misread it, thanks for clarifying!